pkg/packagekit/autoupdates.jsx

```diff
@@ -116,8 +116,7 @@ class DnfImpl extends ImplBase {
     parseCalendar(spec) {
         // see systemd.time(7); we only support what we write, otherwise we treat it as custom config and "unsupported"
         const daysOfWeek = ["mon", "tue", "wed", "thu", "fri", "sat", "sun"];
-        // TODO: allow arbitrary minutes once we support that in the UI widget
-        const validTime = /^((|0|1)[0-9]|2[0-3]):00$/;
+        const validTime = /^((|0|1)[0-9]|2[0-3]):[0-5][0-9]$/;
```

Comment:
Do we support than in UI widget though? :)

Reply:
Yes
![Screen Shot 2021-04-20 at 16 51 51](https://user-images.githubusercontent.com/14921356/115256508-92997f00-a12f-11eb-9fab-0a8b5cbf2f9a.png)

Reply:
Can you also please post screenshot when you save changes?